components/Cma/Schema/index.js

```diff
@@ -245,7 +245,17 @@ function Relationship({ name, schema, required }) {
               {language === 'old-js' ? humps.camelize(name) : name}
             </span>
             {language === 'http' && <span className={s.prefix}>.data</span>}
-            &nbsp;&nbsp;
+            {!isDefinition && (
```

Comment:
Moving the "Required" / "Optional" tag before the type, so that the type will be closer to its inline examples.